sharepoint/sharepoint-ps/sharepoint-pnp/Remove-PnPList.md

```diff
@@ -14,7 +14,10 @@ online version: https://pnp.github.io/powershell/cmdlets/Remove-PnPList.html
 > [!TIP]
 > We encourage you to make improvements to this documentation. Please navigate to https://github.com/pnp/powershell/blob/dev/documentation/Remove-PnPList.md to change this file.
 
-Deletes a list
+Remove-PnPList deletes a list.
+
+###Exceeding the list view threshold
```

Comment:
You'll need to remove the ### per PlatyPS. You can likely just use ## or > [!TIP] etc.